lib/charms/operator_libs_linux/v2/snap.py

```diff
@@ -1014,13 +1043,17 @@ def _wrap_snap_operations(
 
 
 def install_local(
-    filename: str, classic: Optional[bool] = False, dangerous: Optional[bool] = False
+    filename: str,
+    classic: Optional[bool] = False,
+    devmode: Optional[bool] = False,
+    dangerous: Optional[bool] = False,
```

Comment:
I think these should all be changed to non-optional too.

Reply:
While I do agree, the scope of this PR wasn't to refactor the existing code. The other comments were addressed though.

Reply:
Fair -- thanks.